.github/workflows/ci.yaml

```diff
@@ -48,7 +48,28 @@ jobs:
           pushd /tmp/java-compute/google-cloud-compute-small-v1-java
           ./gradlew clean build publishToMavenLocal sourcesJar allJars
           popd
-
+  build-java-21:
+    name: "build(21) except self-service clients"
```

Comment:
Would you add source code comment why build (21) is special?

Reply:
Added.